pkgs/development/php-packages/snuffleupagus/default.nix

```diff
@@ -0,0 +1,41 @@
+{ buildPecl, lib, php, fetchFromGitHub, pcre2 }:
+let
+  pname = "snuffleupagus";
+  version = "0.7.0"; 
+in
+buildPecl {
+  inherit pname version;
+  src = fetchFromGitHub {
+    owner = "jvoisin";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1la6wa9xznc110b7isiy502x71mkvhisq6m8llhczpq4rs4nbcw2";
+  };
+
+  buildInputs = [
+    pcre2
```

Comment:
You probably want to use `pcre'` instead, then it's either `pcre` or `pcre2` depending on PHP version.

Reply:
Where does `pcre'` come from? The variable is undefined when I just replace this...

Reply:
It has to be part of the arguments at the top of this file as well, it's defined in the file php-packages.nix